atasmart

```diff
@@ -56,11 +56,17 @@
 /^208 187/	{print "Reported Uncorrectable Errors", $3; next}
 /^208 188/	{print "Command Timeout", $3; next}
 /^208 189/	{print "High Fly Writes", $3; next}
-/^208 190/	{print "Temperature Difference", $3; next}
+/^208 190/	{
+	print "Temperature Difference", ($3 % 256), "Min", (int($3 / (256*256)) % 256), "Max", int($3 / (256*256*256)); 
+	next
+}
 /^208 191/	{print "G-sense Error Rate", $3; next}
 /^208 192/	{print "Power-off Retract Count", $3; next}
 /^208 193/	{print "Load Cycle Count", $3; next}
-/^208 194/	{print "Temperature", $3; next}
+/^208 194/	{
+	print "Temperature", ($3 % 256), "Min", (int($3 / (256*256)) % 256), "Max", int($3 / (256*256*256*256)); 
```

Comment:
Is there 1 too many *256 for Max?

Reply:
I don't think so.  At least for the drives I've been able to test, 190 is 4 bytes, and 194 is 5 bytes
The pattern seems to be MAX|???|MIN|???|CUR for 194, and MAX|MIN|???|CUR for 190. 
 So the math to extract current and min are the same for both, but the extra 256 is needed to make max work on 194.

```
#smart -x -a 194 da0
0x1300000018
#smart -x -a 190 da0
0x1d160018
#smart -x -a 194 ada0
0x1f0016001b
```

Reply:
OK, works for me. Thanks!